peekaboo/sample.py

```diff
@@ -405,38 +402,10 @@ def file_extension(self):
         return self.__file_extension
 
     @property
-    def mimetypes(self):
```

Comment:
This is a nice simplification. So you're basically migrating the white-/greylisting rules to the new FileToolsReport to simplify Sample here, correct? That's an elegant way to remove some analyser functionality from sample and leverage the new FileTools for that functionality. The code of the rules looks much more explicit to me and the somewhat convoluted mimtypes property of Sample all but disappears. Would this make a useful addition to the commit message?

Reply:
Addressed in #143 by extending the commit message accordingly.